projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-editor.module.ts

```diff
@@ -4,6 +4,7 @@ import { NgModule } from '@angular/core';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { RouterModule } from '@angular/router';
 import { RpxTranslationModule } from 'rpx-xui-translation';
+import { PaletteModule } from '../palette/palette.module';
```

Comment:
Import ordering (why has this been moved)

Reply:
Good spot, @connorpgpmcelroy - it came in from @andywilkinshmcts's work on EXUI-623 but I'll fix it